supabase/migrations/20241104135914_update-attachment-url.sql

```diff
@@ -0,0 +1,18 @@
+UPDATE
+    attachments
+SET
+    url = (
+        SELECT
+            CASE
+                WHEN strpos(a2.url, '?') > 0 THEN SUBSTRING(
+                    a2.url,
+                    strpos(a2.url, '/attachments'),
+                    strpos(a2.url, '?') - strpos(a2.url, '/attachments')
+                )
+                ELSE a2.url
```

Comment:
Couldn't you just do like this?:
`UPDATE attachments SET url = REPLACE(url, 'https://discord.co', '');`

Reply:
I guess I can do that, but this will not replace the params in the url. Don't we want to get rid of them as well? 🤔